infullmvp-kotlin-koin/src/main/java/com/infullmobile/android/infullmvp/InFullMvpPresenter.kt

```diff
@@ -0,0 +1,27 @@
+package com.infullmobile.android.infullmvp
+
+import android.content.Intent
+import android.os.Bundle
+
+abstract class InFullMvpPresenter<PresentedView : MvpView>(var view: PresentedView) {
+
+    open fun onResume() {
+
+    }
+
+    open fun bind(extras: Bundle?, savedInstanceState: Bundle?) {
```

Comment:
This API contract differs from other implementations.

1. Should be abstract
2. extras and savedInstance should not be nullable.

We specifically do not want to surface nullable types at the domain level. We should also force the user to implement this function. Rarely is the case we leave it empty.